lib/helium/label.rb

```diff
@@ -14,14 +14,17 @@ def sensors
 
     def add_sensors(sensors_to_add = [])
       sensors_to_add = Array(sensors_to_add)
+      sensors.add_relationships(sensors_to_add)
+    end
 
-      @client.update_label_sensors(self, sensors: sensors + sensors_to_add)
+    def replace_sensors(sensors_to_replace = [])
+      sensors_to_replace = Array(sensors_to_replace)
```

Comment:
maybe the add/replace/remove functions should handle coercing the input they receive into an Array? that should reduce duplication down the road